nss-ice-phantoms/WorkforceManagement
# Employee list (get all) and Employee create completed

# Description

Issue #1 GET Employees (all) and Issue #2 CREATE Employee completed.  Initial functionality for Issue #3 GET Employee Details implemented.  Still need to add assigned computer and training program information.

## Type of change

# Testing Instructions

1. git fetch --all
2. git checkout rr-employees
3. Run BangazonWorkforce
4. Verify 
    a) Employees nav link navigates to Employees list.  
    b) Create New link on Employees page navigates to the Create Employee form and submitting the form creates a new employee in your database.
    c) Details link by any employee in the list takes you to the details page and displays First Name, Last Name, Full Name and Department Name


# Checklist:
